### DISCUSSION TITLE
el-select中自定义标签怎么使用？

### DISCUSSION THREAD

**Discussion by YHL369z** [element-plus/element-plus#17151] [Q&A]
需要根据数据的type，自定义tag的颜色字体大小。但是#tag和#label一直没有成功，有成功的案例可以提供下方法和element-plus版本配置吗？

**Comment by warmthsea** [comment]
https://element-plus.org/zh-CN/component/select.html#%E8%87%AA%E5%AE%9A%E4%B9%89%E6%A0%87%E7%AD%BE-1
